src/components/HeaderNav/HeaderNav.tsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+import { FaHome, FaMapMarkerAlt, FaQrcode } from 'react-icons/fa';
+
+export function HeaderNav() {
+    return (
+        <nav className="flex gap-6 justify-center bg-secondary rounded p-1">
```

Comment:
```suggestion
        <nav className="flex gap-6 justify-center bg-secondary rounded p-2">
```
Lite mer padding bara